src/loader/x86_64/elf/mod.rs

```diff
@@ -121,7 +154,9 @@ impl Elf {
 impl KernelLoader for Elf {
     /// Loads a kernel from a vmlinux elf image into guest memory.
     ///
-    /// The kernel is loaded into guest memory at offset `phdr.p_paddr` specified by the elf image.
+    /// By default, the kernel is loaded into guest memory at offset `phdr.p_paddr` specified by the elf image.
```

Comment:
Just a style comment: can you please reformat these comment lines to keep them under 100 chars?

Reply:
Fixed, thanks for the heads up. For some reason my editor (VSCode) does not warn me about those.